options/customize.js

```diff
@@ -20,23 +22,50 @@ $(function() {
         "/filters";
 
     var jqxhr = $.ajax({
```

Comment:
@kpeckett This code has been refactored in 414003af1456f049adc214914dfb5928cd7cbdd2.